assets/js/editor-components/default-notice/index.tsx

```diff
@@ -112,6 +98,30 @@ export function DefaultNotice( { block }: { block: string } ) {
 		saveEntityRecord,
 		slug,
 	] );
+	const noticeContent = createInterpolateElement(
+		__(
+			'If you would like to use this block as your default <blockTypeLink/>, <updateLink>update your page settings</updateLink>.',
+			'woo-gutenberg-products-block'
+		),
+		{
+			blockTypeLink: (
+				<span>
+					{ block === 'checkout'
+						? __( 'checkout', 'woo-gutenberg-products-block' )
+						: __( 'cart', 'woo-gutenberg-products-block' ) }
+				</span>
+			),
+			updateLink: (
+				// eslint-disable-next-line jsx-a11y/anchor-is-valid
```

Comment:
It should be noted that initially, this was a link, then we switched to a button due to this problem, we're back again at it being a link. I think it should be fine.

Reply:
Thanks for pointing this out, @senadir. I tried to accomplish the request using a button, but when doing so, the link text was displayed below the regular text and not inline. The only way to get the link inline, was by using [`display: contents`](https://caniuse.com/?search=contents), which is only fully supported by Safari on iOS. That's why I decided to switch back to an anchor element, to display the link as requested in #11777.